boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -14,6 +14,9 @@ config BOOTLOADER_MCUBOOT
 config BOARD_ENABLE_CPUNET
 	default y if !MCUBOOT
 
+config MCUBOOT_USE_ALL_AVAILABLE_RAM
+	default y if BOOTLOADER_MCUBOOT
+
```

Comment:
We should move this to `sdk-nrf` instead. Just define it in some of our kconfig and then make sure to have `depends on` the board you want to enable it for.

Reply:
Some of the features (like defining common static PM configuration, enabling cpunet and MCUboot bootloader) are already added as `[nrf noup]`. If change affect only a specific board in the NCS, I think it may be more reasonable to put it here (it would be way easier for the users to localize source of the change).

Reply:
btw. @sigvartmh, could MCUboot clean up the used non-secure RAM before booting application? That may allow us to prevent leaking data to non-secure code, right ?

Reply:
Sure I'll add one. I was just had a moment of laziness. I thought about then I thought it doesn't make sense that MCUBoot should use more than the available secure RAM unless you are doing something very custom and then your already a little bit on your own.

I'll add the function however this will affect the boot time.

Reply:
I can't view the linked issue, can you explain what enabling `MCUBOOT_USE_ALL_AVAILABLE_RAM` does?

Reply:
`MCUBOOT_USE_ALL_AVAILABLE_RAM ` will make MCUBoot use all the RAM available on the chip. Maybe I should add this wording.

So the way it works after the PR is that MCUboot will have just the `secure_ram` available while before this PR mcuboot would have the start address of the `nonsecure_ram` this could potentially leak data from secure non-aware to non-secure. Since MCUBoot does not clean up residual data in RAM.

Reply:
How does this work with PCD for updating the network core? Not all RAM can be used